src/System.Runtime.Numerics/tests/BigInteger/op_add.cs

```diff
@@ -1,165 +0,0 @@
-            }
-
-            // Add Method - One large and one small BigIntegers
-            for (int i = 0; i < s_samples; i++)
-            {
-                try
-                {
-                    tempByteArray1 = GetRandomByteArray(s_random);
-                    tempByteArray2 = GetRandomByteArray(s_random, 2);
-                    VerifyAdditionString(Print(tempByteArray1) + Print(tempByteArray2) + "b+");
-
-                    tempByteArray1 = GetRandomByteArray(s_random, 2);
-                    tempByteArray2 = GetRandomByteArray(s_random);
-                    VerifyAdditionString(Print(tempByteArray1) + Print(tempByteArray2) + "b+");
-                }
-                catch (IndexOutOfRangeException)
-                {
-                    // TODO: Refactor this
-                    Console.WriteLine("Array1: " + Print(tempByteArray1));
-                    Console.WriteLine("Array2: " + Print(tempByteArray2));
-                    throw;
-                }
-            }
-
-            // Add Method - One large BigIntegers and zero
-            for (int i = 0; i < s_samples; i++)
-            {
-                tempByteArray1 = GetRandomByteArray(s_random);
-                tempByteArray2 = new byte[] { 0 };
-                VerifyAdditionString(Print(tempByteArray1) + Print(tempByteArray2) + "b+");
-
-                tempByteArray1 = new byte[] { 0 };
-                tempByteArray2 = GetRandomByteArray(s_random);
-                VerifyAdditionString(Print(tempByteArray1) + Print(tempByteArray2) + "b+");
-            }
-
-            // Add Method - One small BigIntegers and zero
-            for (int i = 0; i < s_samples; i++)
-            {
-                tempByteArray1 = GetRandomByteArray(s_random, 2);
-                tempByteArray2 = new byte[] { 0 };
-                VerifyAdditionString(Print(tempByteArray1) + Print(tempByteArray2) + "b+");
-
-                tempByteArray1 = new byte[] { 0 };
-                tempByteArray2 = GetRandomByteArray(s_random, 2);
-                VerifyAdditionString(Print(tempByteArray1) + Print(tempByteArray2) + "b+");
-            }
-
-            // 32 bit boundary n1=0 n2=0 c=0
-            VerifyAdditionString("0 0 bAdd");
-
-            // 32 bit boundary n1=0 n2=0 c=1
-            VerifyAdditionString((Math.Pow(2, 31) + Math.Pow(2, 30)) + " " + (Math.Pow(2, 31) + Math.Pow(2, 30)) + " b+");
-
-            // 32 bit boundary n1=0 n2=1 c=0
-            VerifyAdditionString("0" + " " + Math.Pow(2, 32) + " b+");
-
-            // 32 bit boundary n1=0 n2=1 c=1
-            VerifyAdditionString(Math.Pow(2, 31) + " " + (Math.Pow(2, 32) + Math.Pow(2, 31)) + " b+");
-
-            // 32 bit boundary n1=1 n2=0 c=0
-            VerifyAdditionString(Math.Pow(2, 32) + " " + "0" + " b+");
-
-            // 32 bit boundary n1=1 n2=0 c=1
-            VerifyAdditionString((Math.Pow(2, 32) + Math.Pow(2, 31)) + " " + Math.Pow(2, 31) + " b+");
```

Comment:
```
VerifyAdditionString((Math.Pow(2, 32) + Math.Pow(2, 31)) + " " + Math.Pow(2, 31) + " b+");
```
is a duplicate of 
```
VerifyAdditionString(Math.Pow(2, 31) + " " + (Math.Pow(2, 32) + Math.Pow(2, 31)) + " b+");
```
since we now test for identities `x+y===y+x`